src/20_Components/amp-list.html

```diff
@@ -68,6 +98,18 @@
       template="amp-template-id" class="m1">
   </amp-list>
 
+  <!-- ## Using a placeholder animation -->
+  <!--
+    You can use a custom placeholder like an animation to improve the user experience while the list is loading.
```

Comment:
Could we show a skeleton instead of a loading indicator? See https://css-tricks.com/building-skeleton-screens-css-custom-properties/